lambda/templates/repo-config/package.json.hbs

```diff
@@ -1,23 +1,38 @@
 {
   "name": "{{ kebabCase criName }}",
   "description": "",
+  "author": "GOV.UK One Login",
+  "workspaces": [
+    "lambdas/*"
+  ],
   "scripts": {
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "unit": "jest --silent",
     "test": "npm run unit",
     "test:coverage": "npm run unit -- --coverage",
     "pre-commit": "pre-commit run --all-files --show-diff-on-failure",
-    "pc": "npm run pre-commit"
+    "pc": "npm run pre-commit",
+    "sam:validate": "cd infrastructure && sam validate && sam validate --lint",
```

Comment:
Can we add this into a linting task in a GHA?

Reply:
Yes, we should - I'll add it in the PR that touches the workflows later